lib/compiler-llvm/src/translator/code.rs

```diff
@@ -4388,7 +4377,7 @@ impl<'ctx, 'a> LLVMFunctionCodeGenerator<'ctx, 'a> {
                 let input = self.state.pop1()?;
                 let res = self
                     .builder
-                    .build_call(self.intrinsics.sqrt_f32, &[input], "")
+                    .build_call(self.intrinsics.sqrt_f32, &[input.into()], "")
```

Comment:
Does it make sense to change the return value of `pop1` etc.? Do we ever use the non-`into`d version?

Reply:
Actually, we can't. Doing this will just make us move the `into()` calls to other places.

For now, I think we can keep everything as is.